lldb/test/API/lang/swift/async/actors/unprioritised_jobs/TestSwiftActorUnprioritisedJobs.py

```diff
@@ -1,30 +1,54 @@
+import time
+from contextlib import contextmanager
 import lldb
 from lldbsuite.test.decorators import *
 from lldbsuite.test.lldbtest import *
 from lldbsuite.test import lldbutil
 
 
+@contextmanager
+def _managed_async(dbg):
+    async_state = dbg.GetAsync()
+    try:
+        yield
+    finally:
+        dbg.SetAsync(async_state)
+
+
 class TestCase(TestBase):
 
     @swiftTest
-    @skipUnlessFoundation
-    @skipIfWindows  # temporarily skip test until fails can be investigated
-    @skipIfLinux    # temporarily skip test until fails can be investigated
-    @skipIfDarwin   # temporarily skip test until fails can be investigated
     def test_actor_unprioritised_jobs(self):
         """Verify that an actor exposes its unprioritised jobs (queue)."""
         self.build()
-        _, _, thread, _ = lldbutil.run_to_source_breakpoint(
+        _, process, thread, _ = lldbutil.run_to_source_breakpoint(
             self, "break here", lldb.SBFileSpec("main.swift")
         )
         frame = thread.GetSelectedFrame()
+
+        with _managed_async(self.dbg):
+            # Suspend the current thread.
+            thread.Suspend()
+            # Continue - other threads only.
+            self.dbg.SetAsync(True)
+            process.Continue()
+            # Wait - allowing other threads to run.
+            time.sleep(5)
```

Comment:
We usually do something like this (copied from another test)
```
DEFAULT_TIMEOUT = 20 * (10 if ("ASAN_OPTIONS" in os.environ) else 1)
```
to guard against even slower as an bots.

Reply:
Even better would be to wait for something the target actually does, like reading from stdin or creating a file.